components/home/Community.tsx

```diff
@@ -141,24 +92,51 @@ export function Community(): JSX.Element {
     }, []);
 
     return (
-        <section className="bg-gray-50 relative">
-            <div className="w-full mx-auto sm:px-6 lg:px-8 px-10 py-14">
-                <div className="w-full sm:w-1/2 text-center mx-auto mb-11 divide-y divide-gray-400">
-                    <h2 className="text-base font-semibold tracking-wider text-gray-600 uppercase">A GLANCE AT THE</h2>
-                    <p className="mt-3 pt-3 text-5xl font-extrabold text-gray-700">
-                        Community Insights
-                    </p>
-                </div>
-
-                <div className="mx-auto flex flex-col lg:flex-row lg:justify-center divide-y lg:divide-x lg:divide-y-0 divide-gray-500">
+        <section className="flex flex-col lg:flex-row justify-between items-center bg-gray-50 text-blue-dark-contrast">
+            <div className="mt-10 px-page lg:px-24 flex flex-col">
+                <span className="w-24 h-2 bg-teal-light-contrast rounded-full mb-3" />
+                <h1 className="text-5xl pr-2 font-bold">
+                    Community Insights
+                </h1>
+                <p className="text-xl text-gray-800 max-w-prose py-4">
+                    Discover the extensive community behind every FlyByWire Simulations aircraft - a vibrant and active online group who prioritises collaborative work and openness.
```

Comment:
```suggestion
                    Discover the extensive community behind every FlyByWire Simulations aircraft - a vibrant and active online group that prioritises collaborative work and openness.
```
Forgot one last minor grammar fix.